src/components/Main/Schedule/Home.tsx

```diff
@@ -119,6 +82,37 @@ export const Home = () => {
       )}
     </>
   );
+  function onTabClick(value: number) {
```

Comment:
```suggestion
  function onTabClick(tabIdx: number) {
```
would make it a little more obvious